macros/query_tag.sql

```diff
@@ -0,0 +1,40 @@
+{% macro set_query_tag() -%}
```

Comment:
any reason to change the file name? should we update the readme accordingly since it uses `query_tags.sql`? I don't think that should matter, but may be nice for consistency?